doc/articles/features/windows-ui-xaml-window.md

```diff
@@ -4,6 +4,11 @@ uid: Uno.Features.WinUIWindow
 
 # Windows.UI.Xaml.Window
```

Comment:
Just say `Window` here and update to use `Microsoft.UI.Xaml.Window`, as WUX is more or less obsolete